bookkeeper-server/src/main/java/org/apache/bookkeeper/meta/zk/ZKMetadataDriverBase.java

```diff
@@ -260,6 +262,47 @@ public synchronized LedgerManagerFactory getLedgerManagerFactory()
         return lmFactory;
     }
 
+    public CompletableFuture<Void> disableHealthCheck() {
+        CompletableFuture<Void> createResult = new CompletableFuture<>();
+        try {
+            zk.create(getEnableHealthPath(), new byte[0], ZkUtils.getACLs(conf), CreateMode.PERSISTENT);
+            createResult.complete(null);
+        } catch (Exception e) {
+            createResult.completeExceptionally(e);
+        }
+        return createResult;
+    }
+
+    public CompletableFuture<Void>  enableHealthCheck() {
+        CompletableFuture<Void> deleteResult = new CompletableFuture<>();
+
+        try {
+            zk.delete(getEnableHealthPath(), -1);
+            deleteResult.complete(null);
+        } catch (Exception e) {
+            deleteResult.completeExceptionally(e);
+        }
+        return deleteResult;
+    }
+
+    public CompletableFuture<Boolean> isEnableHealthCheck() {
+        CompletableFuture<Boolean> enableResult = new CompletableFuture<>();
+        try {
+            boolean isEnable = (null == zk.exists(getEnableHealthPath(), false));
+            enableResult.complete(isEnable);
+        } catch (Exception e) {
+            enableResult.completeExceptionally(e);
+        }
+        return enableResult;
+    }
+
+
+    public String getEnableHealthPath() {
+        String zkLedgersRootPath = ZKMetadataDriverBase.resolveZkLedgersRootPath(conf);
+        String ledgerParentPath = zkLedgersRootPath.substring(0, zkLedgersRootPath.lastIndexOf("/ledgers"));
+        return String.format("%s/%s", ledgerParentPath, "enableHealthCheck");
```

Comment:
we should add the ZK node name to the BookKeeperConstants class

Reply:
OK,I will fixed